scintillometry/io/psrfits/core.py

```diff
@@ -61,7 +63,37 @@ def open(filename, mode='r', **kwargs):
         # TODO: allow support for more than one HDU.
         if len(reader_list) != 1:
             raise RuntimeError("Current reader can only read one SUBINT HDU.")
+        # TODO Returning one HDU for now. But in the future, we may want to
+        # return a group of it.
         return reader_list[0]
+
+    elif mode in ['w', 'rb+']:
+        # Check if file exists
```

Comment:
Let's not bother for now with `overwrite`, but fail if the file exists.

Reply:
Removed